### ISSUE TITLE
About the loss value

### ISSUE THREAD

**Issue by lushunn** [FlorentF9/DeepTemporalClustering#25]
When I use`python DeepTemporalClustering.py --heatmap False --dist_metric cid  --dataset CBF --pool_size 8`to train the DTC, no matter what dataset used, loss value always  suddenly increases on the 8th epoch, looks like weight of `Lr` and `Lc` also change with epoch, which really confuses me
![746f18f1e6f49fd74b2c6c4de1dfcaf](https://user-images.githubusercontent.com/33921923/149724850-c3c8ed16-1414-433f-8a4a-79e7f73de740.png)
![7676fc6860a4d01398f565615df05a7](https://user-images.githubusercontent.com/33921923/149724860-b0dbf0a6-84d4-4770-b224-835a6388e002.png)
Looking forward to your reply,thank you!

**Comment by sajinpgupta** [comment]
if epoch > self.finetune_heatmap_at_epoch:
            print('On epoch end fine tune heatmap', self.finetune_heatmap_at_epoch)
            K.set_value(self.heatmap_loss_weight, self.final_heatmap_loss_weight)
